src/checkqueue.h

```diff
@@ -130,6 +131,7 @@ class CCheckQueue
     explicit CCheckQueue(unsigned int batch_size, int worker_threads_num)
         : nBatchSize(batch_size)
     {
+        LogInfo("Script verification uses %d additional threads", worker_threads_num);
```

Comment:
Yeah, I thought of different approaches here too, happy to take an alternative here if somebody comes up with something. Otherwise we'll handle this once we get more queues, or a unified thread pool.